app/javascript/components/projects/ProjectsTable.vue

```diff
@@ -44,6 +44,28 @@
       <template #cell(updated_at)="data">
         {{ friendlyDateTime(data.item.updated_at) }}
       </template>
+
+      <template #cell(actions)="data">
+        <RenameProjectModal
+          :project="data.item"
```

Comment:
@rlakey should non-admin users be able to rename a project? currently the back end code only allows admins to rename or remove a project but here a regular user is shown the rename button

Reply:
A Vulcan admin or a project admin?

Reply:
vulcan admin

Reply:
I think we're fine if either a vulcan admin or a project admin can rename.

Reply:
If you want to hide the button for users that aren't admins that's fine too.